### PR TITLE
Divvy credit card section

### PR DESCRIPTION
**Pull request by ialorro**
Adding How to request a Divvy virtual credit card section

### CODE REVIEW

**Comment by ialorro** [comment]
@TQuock added Divvy section in handbook

File: company/how-to-guides-for-staff/how-to-purchase/How to request a Divvy virtual credit card [hunk 1]
```diff
@@ -0,0 +1,31 @@
+## How to request a Divvy virtual credit card
+
+We encourage staff members to not put system or program spend on personal cards or expense reports. We have several processes that can be used to purchase on behalf of Mattermost. For personal purchases, the [How to spend company money](https://handbook.mattermost.com/company/how-to-guides-for-staff/how-to-spend-company-money) policy will be the best place to start. For low dollar system or program charges it's generally appropriate to use Mattermost's virtual card program. For all other purchases, it's recommended to follow the standard [Procurement Process](https://handbook.mattermost.com/company/how-to-guides-for-staff/how-to-purchase/how-to-procure-a-vendor-contract#procuring-a-vendor). If unsure, try the purchases channel on best practices or reach out to Finance.
```
Comment (TQuock) [inline company/how-to-guides-for-staff/how-to-purchase/How to request a Divvy virtual credit card]:
```suggestion
We strongly encourage staff members to not put system or program spend on personal cards or expense reports. We have several processes that can be used to purchase on behalf of Mattermost. For personal purchases, the [How to spend company money](https://handbook.mattermost.com/company/how-to-guides-for-staff/how-to-spend-company-money) policy will be the best place to start. For low dollar system or program charges it's generally appropriate to use Mattermost's virtual card program. For all other purchases, it's recommended to follow the standard [Procurement Process](https://handbook.mattermost.com/company/how-to-guides-for-staff/how-to-purchase/how-to-procure-a-vendor-contract#procuring-a-vendor). If unsure, try the purchases channel on best practices or reach out to Finance.
```

**Review by TQuock — DISMISSED** [review]

**Review by justinegeffen — DISMISSED** [review]

File: company/how-to-guides-for-staff/how-to-purchase/request-a-divvy-virtual-credit-card.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# How to request a Divvy virtual credit card
+
+We strongly encourage staff members not to put system or program spend on personal cards or expense reports. We have several processes that can be used to purchase on behalf of Mattermost. For personal purchases, the [How to spend company money](https://handbook.mattermost.com/company/how-to-guides-for-staff/how-to-spend-company-money) policy will be the best place to start. For low-dollar system or program charges it's generally appropriate to use Mattermost's virtual card program. For all other purchases, it's recommended to follow the standard [Procurement Process](https://handbook.mattermost.com/company/how-to-guides-for-staff/how-to-purchase/how-to-procure-a-vendor-contract#procuring-a-vendor). If you're unsure, try the [Purchases channel](https://community.mattermost.com/private-core/channels/purchases) or reach out to Finance.
+
+Staff members can request a virtual credit card in a couple of ways, depending on the nature of the request.
+
+### No Purchase Order form needed
```
Comment (justinegeffen) [inline company/how-to-guides-for-staff/how-to-purchase/request-a-divvy-virtual-credit-card.md]:
```suggestion
## No Purchase Order form needed
```

File: company/how-to-guides-for-staff/how-to-purchase/request-a-divvy-virtual-credit-card.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+Staff members can make a request by emailing AP@mattermost.com and provide the following information on the email:
+
+* Vendor name
+* Purpose of the service
+* Service period
+* Cost either monthly or annually
+* MLT Leader approving the request (MLT Leader should also be CC’d on the email request)
+* Vendor billing contact (if any)
+
+### Purchase Order form required
```
Comment (justinegeffen) [inline company/how-to-guides-for-staff/how-to-purchase/request-a-divvy-virtual-credit-card.md]:
```suggestion
## Purchase Order form required
```

**Review by justinegeffen — APPROVED** [review]

**Review by TQuock — COMMENTED** [review]
looks good to me :)

**Comment by ialorro** [comment]
Can we please make this section visible in the handbook? Thank you!

**Pull request merged**